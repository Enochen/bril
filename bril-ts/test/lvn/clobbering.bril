@main {
  a: int = const 4;
  b: int = const 2;
  sum1: int = add b a;
  prod1: int = mul a b;
  prod2: int = mul b a;
  print prod2;
  prod2: int = mul prod1 sum1;
  print prod2;
}
