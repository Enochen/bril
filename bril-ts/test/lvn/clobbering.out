@main {
  prod1: int = const 8;
  print prod1;
  prod2: int = const 48;
  print prod2;
}
